src/main/java/appeng/client/render/tesr/CableBusFastTESR.java

```diff
@@ -0,0 +1,306 @@
+
+			for( BakedQuad q : quads )
+			{
+				q.pipe( l );
+			}
+		} );
+	}
+
+	private static CableBusRenderState getRenderingState( final IBlockAccess world, BlockPos pos )
+	{
+		IBlockState blockState = world.getBlockState( pos );
+		try
+		{
+			blockState = blockState.getActualState( world, pos );
+		}
+		catch( Exception e )
+		{
+			// ignore
+		}
+
+		try
+		{
+			blockState = blockState.getBlock().getExtendedState( blockState, world, pos );
+		}
+		catch( Exception e )
+		{
+			// ignore
+		}
+
+		if( blockState == null || !( blockState instanceof IExtendedBlockState ) )
+		{
+			return null;
+		}
+
+		IExtendedBlockState extendedBlockState = (IExtendedBlockState) blockState;
+		return extendedBlockState.getValue( BlockCableBus.RENDER_STATE_PROPERTY );
+	}
+
+	private List<BakedQuad> addFacade( BlockRenderLayer layer, Map<EnumFacing, FacadeRenderState> facades, EnumFacing side, AEAxisAlignedBB busBounds, boolean thinFacades, long rand )
+	{
+		ArrayList<BakedQuad> ret = new ArrayList<>();
+
+		CubeBuilder builder = new CubeBuilder( DefaultVertexFormats.ITEM, ret );
+		FacadeRenderState facadeState = facades.get( side );
+		IBlockState blockState = facadeState.getSourceBlock();
+
+		builder.setDrawFaces( EnumSet.allOf( EnumFacing.class ) );
+
+		// Reset to no color multiplicator
+		builder.setColorRGB( 0xFFFFFF );
+		builder.useStandardUV();
+
+		// Do not add the translucent facade in any other layer than translucent
+		boolean translucent = AEApi.instance().partHelper().getCableRenderMode().transparentFacades;
+		if( translucent && layer != BlockRenderLayer.TRANSLUCENT )
+		{
+			return ret;
+		}
+
+		final float thickness = thinFacades ? 1 : 2;
+
+		BlockRendererDispatcher blockRendererDispatcher = Minecraft.getMinecraft().getBlockRendererDispatcher();
+		IBakedModel blockModel = blockRendererDispatcher.getModelForState( blockState );
+
+		final int color;
+		if( translucent )
+		{
+			color = 0x4CFFFFFF;
+		}
+		else
+		{
+			color = 0xFFFFFFFF;
+		}
+
+		// TODO: Cache this
+		for( EnumFacing facing : EnumFacing.values() )
+		{
+			TextureAtlasAndTint spriteAndTint = FacadeBuilder.getSprite( blockModel, blockState, facing, rand );
+			if( spriteAndTint != null && spriteAndTint.getSprite() != null )
+			{
+				// Use the tint color from the item stack here, which is based upon the assumption that the
+				// model used for the block will use the same meaning for tint indices as the item model does
+				if( spriteAndTint.getTint() != -1 )
+				{
+					int tintColor = facadeState.resolveTintColor( spriteAndTint.getTint() );
+
+					// Still apply the transparency color
+					tintColor &= 0xFFFFFF;
+					tintColor |= color & 0xFF000000;
+					builder.setColor( tintColor );
+				}
+				else
+				{
+					builder.setColor( color );
+				}
+
+				builder.setTexture( facing, spriteAndTint.getSprite() );
+			}
+			else
+			{
+				builder.setColor( color );
+			}
+		}
+
+		// builder.setDrawFaces( facadeState.getOpenFaces() );
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') This block of commented-out lines of code should be removed. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonar.orod.org/coding_rules#rule_key=squid%3ACommentedOutCodeLine)